extensions/smallrye-jwt/deployment/pom.xml

```diff
@@ -48,7 +48,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
+            <artifactId>quarkus-resteasy-reactive-jsonb-deployment</artifactId>
```

Comment:
Not sure about this change - it is still expected to work with Resteasy Classic

Reply:
Currently reactive stuff is not tested with `smallrye-jwt`, however classic is tested with `quarkus-integration-test-smallrye-jwt-token-propagation` and `quarkus-integration-test-smallrye-jwt-oidc-webapp`. I thought it's a good thing to know both reactive and classic work :-). I can change it if you like, but than I don't know where to put the test that actually verifies the issue.

Reply:
@michalvavrik But does replacing a `jsonb` dependency forces the use of Resteasy Reactive in `quarkus-integration-test-smallrye-jwt-token-propagation` ?

Reply:
@sberyozkin for `test` scope in `extension/smallrye-jwt` I replaced `quarkus-resteasy-jsonb-deployment` with `quarkus-resteasy-reactive-jsonb-deployment`, so should have no impact in integration modules as test dependencies are not transitive.

I'm not sure if there is not misunderstanding between us - I don't say certain module _must_ use RR, I just say there are 3 modules testing `smallrye-jwt` and if one of them had RR, I could add there a test and in general, I don't why should classic be preferred over reactive.

Reply:
@michalvavrik Oh sorry, I missed it was a test scope.